src/app/public/modules/angular-tree/angular-tree-node.component.html

```diff
@@ -0,0 +1,115 @@
+<div
+  role="treeitem"
+  [attr.aria-current]="node.isActive"
+  [attr.aria-expanded]="node.isExpanded"
+  [attr.aria-selected]="ariaSelected()"
+  [class]="node.getClass()"
+  [class.tree-node]="true"
+  [class.tree-node-active]="node.isActive"
+  [class.tree-node-collapsed]="node.isCollapsed && node.hasChildren"
+  [class.tree-node-expanded]="node.isExpanded && node.hasChildren"
+  [class.tree-node-focused]="node.isFocused"
+  [class.tree-node-leaf]="node.isLeaf"
+>
+
+  <tree-node-drop-slot *ngIf="index === 0"
+    [dropIndex]="node.index"
+    [node]="node.parent"
+  >
+  </tree-node-drop-slot>
+
+  <!-- #region <tree-node-wrapper> custom template --->
+  <div
+    class="node-wrapper"
+    [class.sky-angular-tree-node-active]="showActiveClass()"
+    [class.sky-angular-tree-node-selected]="showSelectedClass()"
+    [style.padding-left]="node.getNodePadding()"
+  >
+
+    <!-- #region <tree-node-expander> custom template --->
+    <div class="sky-toggle-children-wrapper">
+      <button *ngIf="node.hasChildren"
+        class="sky-btn sky-btn-link sky-toggle-children"
+        tabindex="-1"
+        type="button"
+        [attr.aria-label]="'skyux_angular_tree_click_to_expand' | skyLibResources"
+        [class.toggle-children-wrapper-expanded]="node.isExpanded"
+        [class.toggle-children-wrapper-collapsed]="node.isCollapsed"
+        (click)="node.mouseAction('expanderClick', $event)"
+      >
+        <sky-icon
+          [icon]="node.isExpanded ? 'chevron-down' : 'chevron-right'"
+          size="sm"
+        >
+        </sky-icon>
+      </button>
+      <span *ngIf="!node.hasChildren"
+        class="toggle-children-placeholder"
+      >
+      </span>
+    </div>
+    <!-- #endregion <tree-node-expander> --->
+
+    <!-- Note: we've made changes to the click event, tabindex, and classes when compared to the original template. -->
+    <!-- https://github.com/500tech/angular-tree-component/blob/master/lib/components/tree-node-wrapper.component.ts -->
+    <div
+      class="node-content-wrapper"
+      [allowDragoverStyling]="node.allowDragoverStyling()"
+      [attr.tabindex]="tabIndex"
+      [treeAllowDrop]="node.allowDrop"
+      [treeDrag]="node"
+      [treeDragEnabled]="node.allowDrag()"
+      (contextmenu)="node.mouseAction('contextMenu', $event)"
+      (dblclick)="node.mouseAction('dblClick', $event)"
+      (focus)="onFocus()"
+      (keydown.ArrowLeft)="onArrowLeft($event)"
+      (keydown.ArrowRight)="onArrowRight($event)"
```

Comment:
Each node will have two keydown events; might be better to consolidate into one and react to the key code.

Reply:
(For performance reasons.)

Reply:
This was taken from the library's template (see previous comment). Is it better to keep it as close to the same or be more performant?

Reply:
Also, if we do make a change - remember that `keyCode` has been deprecated in favor of the `key` property.

Reply:
It would be my preference to combine them into one event to be more performant.

Reply:
We talked offline about this. We'll keep it as two separate events for the sake of time, but we may need to come back to it if performance becomes a problem.